packages/core/src/components/forms/radioGroup.tsx

```diff
@@ -49,7 +49,7 @@ export interface IRadioGroupProps extends IProps {
     options?: IOptionProps[];
 
     /** Value of the selected radio. The child with this value will be `:checked`. */
-    selectedValue?: string;
+    selectedValue?: string | number;
```

Comment:
remove unrelated change

Reply:
oh but it is related, or i wouldn't have pushed it. for `FormatSelect`, which uses the index in an array. `Radio` supports number values, so `RadioGroup` should do.

Reply:
separate PR please